index.html

```diff
@@ -48,198 +61,254 @@ <h2 class="describe">I am a Software Developer</h2>
-    <a class="bgimage"></a>
-    <h3 class="Stories">Multi-Post Stories</h3>
-    <p class="work-words">
-      A daily selection of privately personalized reads; no accounts or sign-ups
-      required. has been the industry's standard dummy text ever since the
-      1500s, when an unknown printer took a standard dummy text.
-    </p>
+    <div class="recent-works-head">
+      <h2 class="recent-works">My Recent Works</h2>
+      <hr />
+    </div>
 
-    <ul class="languages">
-      <li class="language-icons">
-        <a href="#"><img src="images/css.png" alt="" /></a>
-      </li>
-      <li class="language-icons">
-        <a href="#"><img src="images/html.png" alt="" /></a>
-      </li>
-      <li class="language-icons">
-        <a href="#"><img src="images/bootstrap.png" alt="" /></a>
-      </li>
-      <li class="language-icons">
-        <a href="#"><img src="images/ruby.png" alt="" /></a>
-      </li>
-      <br />
-    </ul>
+    <div class="recent-works-words">
+      <img class="bgimage" src="images/desktop/works.png" alt=""/>
+      <article class="stories-words">
+        <h3 class="Stories">Multi-Post Stories</h3>
+        <p class="work-words">
+          A daily selection of privately personalized reads; no accounts or sign-ups
+          required. has been the industry's standard dummy text ever since the
+          1500s, when an unknown printer took a standard dummy text.
+        </p>
 
-    <li>
-      <a class="button" href="#">See Project</a>
-    </li>
+        <ul class="languages">
+          <li class="language-icons">
+            <a href="#"><img src="images/css.png" alt="" /></a>
+          </li>
+          <li class="language-icons">
+            <a href="#"><img src="images/html.png" alt="" /></a>
+          </li>
+          <li class="language-icons">
+            <a href="#"><img src="images/bootstrap.png" alt="" /></a>
+          </li>
+          <li class="language-icons">
+            <a href="#"><img src="images/ruby.png" alt="" /></a>
+          </li>
+          <br />
+        </ul>
+
+        <li>
+          <a class="button" href="#">See Project</a>
+        </li>
+      </article>
+    </div>
   </section>
 
-  <section class="Professional">
-    <h5 class="heading5">Profesional Art Printing Data</h5>
-    <p class="art-words">
-      A daily selection of privately personalized reads; no accounts or sign-ups
-      required. has been the industry's standard
-    </p>
-    <a class="bgimage2"></a>
-    <ul class="f-item">
-      <li>
-        <a class="icon2" href="#"><img src="images/html1.png" alt="" /></a>
-      </li>
-      <li>
-        <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
-      </li>
-      <li>
-        <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
-      </li>
-    </ul>
 
-    <li>
-      <a class="button2" href="#">See Project</a>
-    </li>
-  </section>
+  <div class="Professional-cards">
+    <section class="Professional">
```

Comment:
- [ ] Kindly update the project card according to the design. The `See Project` button should not be visible at first sight. Please make it so that the dark opacity disappears on hover and the `See Project` button appears. Kindly check the below Figma design as a reference. ✍️
**Please implement it on all project cards.**

**Normal State**
<img src="https://user-images.githubusercontent.com/17243325/208041326-c8b1b1b5-93d1-4d2f-b8eb-1640160c0891.png" width="300">

**Hover State**
<img src="https://user-images.githubusercontent.com/17243325/208041421-61a2ac78-056c-4d52-b8f2-7c4db16fe908.png" width="300">